### ISSUE TITLE
openwrt with gstreamer 1.4.5

### ISSUE THREAD

**Issue by jiqianghu** [hzeller/gmrender-resurrect#130]
gmediarender: output_gstreamer.c: 503: output_gstreamer_init: Assertion `player_ != ((void *)0)' failed.
Aborted

**Comment by jhpark555** [comment]
I have a same error like below.

root@OpenWrt:~# gmediarender
gmediarender 0.0.7-git started [ gmediarender 2017-07-16_aef6a4154e (libupnp-1.6.19; glib-2.53.7; gstreamer-1.8.2) ].
Logging switched off. Enable with --logfile=<filename> (e.g. --logfile=/dev/stdout for console)
Assertion failed: player_ != NULL (output_gstreamer.c: output_gstreamer_init: 503)
Aborted.

**Comment by hzeller** [comment]
did you install all the necessary gstreamer plugins ?

**Comment by jhpark555** [comment]
Hi Hzeller,

After I installed all the module related with it, the error was gone. then I tried to cast internet radio in the Foobar on the device, the error happens again like below.   

Thanks.


gmediarender 0.0.7-git started [ gmediarender 2017-07-16_aef6a4154e (libupnp-1.6.19; glib-2.53.7; gstreamer-1.8.2) ].
Logging switched off. Enable with --logfile=<filename> (e.g. --logfile=/dev/stdout for console)
Ready for rendering.
ERROR [2017-09-13 00:46:45.159094 | gstreamer] source: Error: Could not establish connection to server. (Debug: gstso
uphttpsrc.c(1573): gst_soup_http_src_parse_status (): /GstPlayBin:play/GstURIDecodeBin:uridecodebin1/GstSoupHTTPSrc:s
ource:
Could not connect: Host is unreachable (4), URL: http://192.168.101.105:56923/content/f7cf6ad9c8544ea51228f4804a467b2
3.wav?profile_id=0&convert=wav, Redirect to: (NULL))
ERROR [2017-09-13 00:46:45.160053 | gstreamer] source: Error: Could not establish connection to server. (Debug: gstso
uphttpsrc.c(1573): gst_soup_http_src_parse_status (): /GstPlayBin:play/GstURIDecodeBin:uridecodebin1/GstSoupHTTPSrc:s
ource:
Could not connect: Host is unreachable (4), URL: http://192.168.101.105:56923/content/f7cf6ad9c8544ea51228f4804a467b2
3.wav?profile_id=0&convert=wav, Redirect to: (NULL))
ERROR [2017-09-13 00:46:45.161409 | gstreamer] source: Error: Internal data flow error. (Debug: gstbasesrc.c(2948): g
st_base_src_loop (): /GstPlayBin:play/GstURIDecodeBin:uridecodebin1/GstSoupHTTPSrc:source:
streaming task paused, reason error (-5))
ERROR [2017-09-13 00:46:45.162082 | gstreamer] typefindelement1: Error: Stream doesn't contain enough data. (Debug: g
sttypefindelement.c(983): gst_type_find_element_chain_do_typefinding (): /GstPlayBin:play/GstURIDecodeBin:uridecodebi
n1/GstTypeFindElement:typefindelement1:
Can't typefind stream)
